dolphinscheduler-dist/src/main/assembly/assembly-plugins.sh

```diff
@@ -36,12 +38,16 @@ storage-plugins
 task-plugins
 )
 
-for plugin_path in ${PLUGINS_PATH[@]}
-do
-  cd $BIN_DIR/plugins/$plugin_path
-  find ./* -name "*.jar" | xargs -I {} mv {} ./
-  ls -d */ | xargs -I {} rm -rf {}
-done
+if [ $PLUGINS_ASSEMBLY == "true" ]; then
+  for plugin_path in ${PLUGINS_PATH[@]}
+  do
+    cd $BIN_DIR/plugins/$plugin_path
+    find ./* -name "*.jar" | xargs -I {} mv {} ./
+    ls -d */ | xargs -I {} rm -rf {}
+  done
+else
+  rm -rf $BIN_DIR/plugins/*
+fi
```

Comment:
Can we use env or jvm -D args to control this, then we can remove staging profile.

And why we set this true in staging, since we will not upload in test?

Reply:
> Can we use env or jvm -D args to control this, then we can remove staging profile.

Good idea. Will fix it.